### PR TITLE
[ch17639] Add connector for pollution incidences

### PR DESCRIPTION
**Pull request by robgc**
[ch17639]
**Important**: The data source URL needs to be specified later

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,13 +15,14 @@ The following connectors are available:
...
+- pollutionIncidences
 
 ## Testing connectors
 
 You can test in your local environment with this CLI tool:
 
 ```
-node run_connector.js -c [cameras|incidences|intensitylines|intensitypois|servicelevel|trafficlightsacustw|trafficlightsred|streetsgeocoder]
+node run_connector.js -c [cameras|incidences|intensitylines|intensitypois|servicelevel|trafficlightsacustw|trafficlightsred|streetsgeocoder|pollutionIncidences]
 ```
```
Comment (cayetanobv) [inline README.md]: lower case in CLI name

File: README.md [hunk 2]
```diff
@@ -15,13 +15,14 @@ The following connectors are available:
...
 - streetsgeocoder (inverse geocoding for incidences)
+- pollutionIncidences
 
 ## Testing connectors
 
 You can test in your local environment with this CLI tool:
 
 ```
-node run_connector.js -c [cameras|incidences|intensitylines|intensitypois|servicelevel|trafficlightsacustw|trafficlightsred|streetsgeocoder]
+node run_connector.js -c [cameras|incidences|intensitylines|intensitypois|servicelevel|trafficlightsacustw|trafficlightsred|streetsgeocoder|pollutionIncidences]
```
Comment (cayetanobv) [inline README.md]: lower case in CLI name

File: run_connector.js [hunk 3]
```diff
@@ -27,6 +28,8 @@ function getTrafficConnector(connector) {
       return new TrafficLightsRed();
     case 'streetsgeocoder':
       return new StreetsGeocoder();
+    case 'pollutionIncidences':
```
Comment (cayetanobv) [inline run_connector.js]: lower case in CLI name

File: README.md (hunk 2 above)
Comment (robgc) [inline README.md]:
```suggestion
node run_connector.js -c [cameras|incidences|intensitylines|intensitypois|servicelevel|trafficlightsacustw|trafficlightsred|streetsgeocoder|pollutionincidences]
```

File: run_connector.js (hunk 3 above)
Comment (robgc) [reply run_connector.js]:
```suggestion
    case 'pollutionincidences':
```

**Pull request merged**